concent_api/core/tests/test_core_views.py

```diff
@@ -480,34 +481,31 @@ def test_send_should_return_http_202_if_task_to_compute_deadline_is_correct(self
             compute_task_def['task_id']     = str(i)
             compute_task_def['subtask_id']  = str(i)
             compute_task_def['deadline']    = deadline
-            task_to_compute                 = message.TaskToCompute(
+
+            deserialized_task_to_compute = self._get_deserialized_task_to_compute(
                 compute_task_def     = compute_task_def,
-                provider_public_key  = PROVIDER_PUBLIC_KEY,
-                requestor_public_key = REQUESTOR_PUBLIC_KEY,
+                provider_public_key  = self.PROVIDER_PUBLIC_KEY,
+                requestor_public_key = self.REQUESTOR_PUBLIC_KEY,
             )
 
-            serialized_task_to_compute   = dump(task_to_compute,             REQUESTOR_PRIVATE_KEY,   PROVIDER_PUBLIC_KEY)
-            deserialized_task_to_compute = load(serialized_task_to_compute,  PROVIDER_PRIVATE_KEY,  REQUESTOR_PUBLIC_KEY, check_time = False)
-
-            with freeze_time("2017-11-17 10:00:00"):
-                report_computed_task = message.tasks.ReportComputedTask(
-                    task_to_compute = deserialized_task_to_compute
-                )
-                force_report_computed_task = message.ForceReportComputedTask(
-                    report_computed_task = report_computed_task
-                )
+            report_computed_task = message.tasks.ReportComputedTask(
+                task_to_compute = deserialized_task_to_compute
+            )
+            force_report_computed_task = message.ForceReportComputedTask(
```

Comment:
you can extract it to a function / method and use throughout tests, to avoid duplication

Reply:
Done

Reply:
Done